source/databricks/calculation_engine/package/datamigration/migration_scripts/202405071800_Modify_settlement_report_metering_point_time_series_v1_view.sql

```diff
@@ -0,0 +1,21 @@
+DROP VIEW IF EXISTS {SETTLEMENT_REPORT_DATABASE_NAME}.metering_point_time_series_v1
+GO
+
+CREATE VIEW IF NOT EXISTS {SETTLEMENT_REPORT_DATABASE_NAME}.metering_point_time_series_v1 as
+SELECT m.calculation_id,
+       m.metering_point_id,
+       m.metering_point_type,
+       m.resolution,
+       m.grid_area_code,
+       m.energy_supplier_id,
+       DATE_TRUNC('day', FROM_UTC_TIMESTAMP(t.observation_time, 'Europe/Copenhagen')) AS observation_day,
+       ARRAY_SORT(ARRAY_AGG(struct(t.observation_time, t.quantity)))                  AS quantities
+FROM {BASIS_DATA_DATABASE_NAME}.metering_point_periods AS m
+         INNER JOIN (SELECT calculation_id FROM {BASIS_DATA_DATABASE_NAME}.calculations) AS c ON c.calculation_id = m.calculation_id
```

Comment:
Can this be simplified to...?
```suggestion
         INNER JOIN {BASIS_DATA_DATABASE_NAME}.calculations AS c ON c.calculation_id = m.calculation_id
```

Reply:
should also work